### PR TITLE
ref: revise setup.py, remove stray license headers

### PR DESCRIPTION
**Pull request by joshuarli**
Just a few small things, see commit messages.

Also removed some stray license headers missed by https://github.com/getsentry/sentry/pull/14095.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -39,7 +39,8 @@
 
 ROOT = os.path.realpath(os.path.join(os.path.dirname(sys.modules["__main__"].__file__)))
 
-# Add Sentry to path so we can import distutils
+# add sentry to path so we can import distutils
+# XXX: consequentially, this means sentry must be pip installed with --no-use-pep517
```
Comment (dcramer) [inline setup.py]: This was just to make life easier. Wasn’t ever required iirc.

**Review by dcramer — APPROVED** [review]

File: setup.py [hunk 2]
```diff
@@ -19,18 +19,18 @@
 Python, but it contains a full API for sending events from any language, in
 any application.
 
-:copyright: (c) 2011-2014 by the Sentry Team, see AUTHORS for more details.
-:license: BSD, see LICENSE for more details.
+:copyright: (c) 2011-2019 by the Sentry Team, see AUTHORS for more details.
+:license: BSL-1.1, see LICENSE for more details.
```
Comment (EvanPurkhiser) [inline setup.py]: Can you just delete this part tbh, we removed this copyright / license header everywhere else

Comment (joshuarli) [reply setup.py]: I felt like the setup.py was a special case, but yeah, I can totally remove this entire docstring altogether. We already have the license in setup args.

Comment (BYK) [reply setup.py]: We can remove this as it is covered below and in the repo directly.

File: setup.py [hunk 3]
```diff
@@ -1,36 +1,13 @@
...
-"""
 from __future__ import absolute_import
 
-# if sys.version_info[:2] != (2, 7):
-#     print 'Error: Sentry requires Python 2.7'
-#     sys.exit(1)
+import sys
+
+if sys.version_info[:2] != (2, 7):
+    sys.exit("Error: Sentry requires Python 2.7.")
```
Comment (BYK) [inline setup.py]: 😿

File: setup.py (hunk 1 above)
Comment (BYK) [reply setup.py]: If it works without this, we should 100% remove this as it complicates things a bit too much.

File: setup.py [hunk 4]
```diff
@@ -48,23 +26,9 @@
     BuildJsSdkRegistryCommand,
 )
 
-# The version of sentry
```
Comment (BYK) [inline setup.py]: image

File: setup.py [hunk 5]
```diff
@@ -48,23 +26,9 @@
...
-# http://www.eby-sarna.com/pipermail/peak/2010-May/003357.html)
-for m in ("multiprocessing", "billiard"):
-    try:
-        __import__(m)
-    except ImportError:
-        pass
-
 IS_LIGHT_BUILD = os.environ.get("SENTRY_LIGHT_BUILD") == "1"
 
-# we use pip requirements files to improve Docker layer caching
```
Comment (BYK) [inline setup.py]: I think this may stay? At least until we move to poetry? :trollface:

File: src/sentry/conf/urls.py [hunk 6]
```diff
@@ -1,12 +1,3 @@
-"""
-sentry.conf.urls
-~~~~~~~~~~~~~~~~
-
-These are additional urls used by the Sentry-provided web server
-
-:copyright: (c) 2012 by the Sentry Team, see AUTHORS for more details.
```
Comment (BYK) [inline src/sentry/conf/urls.py]: Let me quote the PR title:
>ref: revise setup.py

This file says `src/sentry/conf/urls.py` - I sense contradictions in you but the force is strong.

**Review by BYK — APPROVED** [review]
Nicely done!

File: src/sentry/conf/urls.py (hunk 6 above)
Comment (joshuarli) [reply src/sentry/conf/urls.py]: scope creep gets the best of us, haha

File: setup.py (hunk 3 above)
Comment (joshuarli) [reply setup.py]: 😿

File: setup.py (hunk 5 above)
Comment (joshuarli) [reply setup.py]: The future is mysterious and hopefully bright!

**Pull request merged**